app/src/components/report/MyModal.tsx

```diff
@@ -0,0 +1,83 @@
+import React, { useState, useImperativeHandle, forwardRef } from 'react';
+import Box from '@mui/material/Box';
+import { Button } from 'react-bootstrap';
+import Typography from '@mui/material/Typography';
+import Modal from '@mui/material/Modal';
+
+const style = {
+    position: 'absolute' as 'absolute',
```

Comment:
If you declare the style variable as `const STYLE: SxProps = { ... }`, you won't need the awkward `'absolute' as 'absolute'` :)